src/pages/steps/AdditionalInformationStep/FormScore.tsx

```diff
@@ -242,8 +242,8 @@ const FormScore = ({ completedFields, offerId, scope }: Props) => {
   }, [score, minimumScore]);
 
   const tipField = useMemo(() => {
-    if (score === 100)
-      return t(`create.additionalInformation.event_score.tip.completed`);
+    if (score === 100) return;
```

Comment:
Removed this since `tipField` is only rendered when we have tips to give, ie. when the score isn't 100